sdk/eventgrid/azure-eventgrid/swagger/README.PYTHON_T2.md

```diff
@@ -13,11 +13,10 @@ no-namespace-folders: true
 output-folder: ../azure/eventgrid/_generated
 source-code-folder-path: ./azure/eventgrid/_generated
 input-file: 
-    - https://raw.githubusercontent.com/t-swpill/azure-rest-api-specs/add-cloud-event-publish-to-event-grid/specification/eventgrid/data-plane/Microsoft.EventGrid/stable/2018-01-01/EventGrid.json
+    - https://github.com/ellismg/azure-rest-api-specs/blob/4bb5b76cb8401896b15f1be3fdaac6bd5d299b17/specification/eventgrid/data-plane/Microsoft.EventGrid/stable/2018-01-01/EventGrid.json
```

Comment:
If I were to do this I'd use a core azure hash, but I'm absolutely nitpicking now just because I saw this comment.  Do as you see fit, but keep in mind the above as an option going forward.